data/campaigns/Eastern_Invasion/ART_MUSIC_LICENSE.txt

```diff
@@ -0,0 +1,298 @@
+
+AUDIO
+===============
+
+MUSIC
+---------------
+
+cry from elensefar (pluft) - CC BY-SA 4.0
+    https://forums.wesnoth.org/viewtopic.php?t=48565
+    https://soundcloud.com/pluft/cry-from-elensefar
+
+heavens (gabriel silver), remixed by dalas - CC BY-SA 4.0 (both)
+    https://forums.wesnoth.org/viewtopic.php?t=51483
+    https://www.youtube.com/watch?v=AOvwWOGH7bQ&list=PLk-SdLbgd5ol62NX0q7ys5ZbmJFVAMzCF&index=2
+
+
+SOUNDS
+---------------
+
+wind (hackerb9/Tony B kksm) - CC 0
+    edited by dalas; I release the edited sound under CC BY-SA 4.0
+    https://freesound.org/people/hackerb9/sounds/555246/
+
+gate-fall
+lich_die_wail
+magic-twilight
+magic-twilight-miss
+    made by dalas by modifying pre-existing wesnoth sounds (GPL v2.0 or later)
+
+warhammer - GPL v2.0 or later
+    made by nemaara using pre-existing wesnoth sounds
+
+ASSORTED IMAGES
+===============
+
+ATTACKS
+---------------
+
+blast-wave
+sword-astral-short
+sword-orcish-flaming
+    made by dalas by modifying pre-existing wesnoth images (GPL v2.0 or later)
+
+sword-orcish-swing
+    from the Reign of Lords add-on (GPL v2.0 or later)
+
+
+HALO
+---------------
+
+terror - CC BY-SA 4.0
+    created by dalas
+
+idle-flash
+fire-aura-small
+light-shield
+    made by dalas by modifying pre-existing wesnoth images (GPL v2.0 or later)
+
+shadow-mage-halo - from the Liberty campaign (GPL v2.0 or later)
+
+
+ICONS
+---------------
+
+banana
+drowning
+    made by dalas (CC BY-SA 4.0)
+
+coins
+ninja
+ogre
+    made by dalas by modifying pre-existing wesnoth images (GPL v2.0 or later)
+
+planb (Q_x), modified by dalas - GPL 2.0 and GPL 3.0
+    https://opengameart.org/content/dwarven-map
+    source file: https://github.com/wesnoth/resources/blob/master/campaigns/Eastern_Invasion/images/plan.psd
+
+skull.png (Atmostatic), modified by dalas - GPL 2.0 and GPL 3.0
+    https://opengameart.org/content/skull-1
+
+
+ITEMS
+---------------
+
+altar-evil-amulet
+crystal-quiver
+horse-cage
+gohere-empty
+plauge-staff
+rider-corpse
+secret-item
+shield-of-the-sentinel
+shield-of-the-sentinel-stand
+swamp_herb
+whitefang-flag
+    all either pre-existing wesnoth content or made by dalas by modifying pre-existing wesnoth images (GPL v2.0 or later)
+
+ant-ambrosia (admurin), modified by dalas - CC BY-SA 4.0
+    https://opengameart.org/content/admurins-insect-items
+
+yeti meat (bluecarrot16), modified by dalas - GPL 3.0
+    https://opengameart.org/content/lpc-food
+    See CREDITS-food.txt in the zip through the URL
```

Comment:
If this is just one item from one of those zip files, it should have the details of that individual item.

This might (or might not) solve another issue - I don't think we accept "GPLv3 only". We accept GPLv2+, which gives the recipient the option of choosing to redistribute under GPLv3, but also allows them to redistribute under GPLv2.